draft-ietf-core-sid.md

```diff
@@ -147,16 +147,16 @@ guarantee their uniqueness. SIDs are registered in blocks called "SID ranges".
 SIDs are assigned permanently.
 Items introduced by a new revision of a YANG
 module are added to the list of SIDs already assigned.
-Assignment of SIDs to YANG items SHOULD be automated.
-For more details how this can be achieved, and when manual interventions may be appropriate, see {{sid-auto-generation}}.
+Assignment of SIDs to YANG items can be automated using the methods from {{sid-auto-generation}}.
```

Comment:
Hmm, we already showed the previous change to the IESG and a DISCUSS got cleared, IIRC.  Why are we changing this again?